### PR TITLE
Port OLKB Preonic to VIAL

### PR DESCRIPTION
**Pull request by alderaan-rock**
Hello, 
 
I copied the default keymap and removed two layers (COLEMAK & DVORAK), as they were preventing the firmware from compiling and were not necessary.

Have a nice day !

### CODE REVIEW

File: keyboards/preonic/keymaps/vial/readme.md
Comment (lesshonor) [inline keyboards/preonic/keymaps/vial/readme.md:1]: I'd delete this file; it doesn't really add anything.

**Review by lesshonor — CHANGES_REQUESTED** [review]
Which revision of the Preonic is this for?
They aren't all wired the same way, so they can't have a single Vial keymap. You'll need to move this under whatever revision folder you tested it with.

File: keyboards/preonic/keymaps/vial/readme.md
Comment (alderaan-rock) [reply keyboards/preonic/keymaps/vial/readme.md:1]: I deleted the file as suggested

**Comment by alderaan-rock** [comment]
Hi,

I moved the Vial keymap to preonic/rev3_drop/keymaps/vial 

Regards,

**Pull request merged**